app/models/setting/discovered.rb

```diff
@@ -16,6 +16,7 @@ def self.load_defaults
     Setting.transaction do
       [
         self.set('discovery_fact', N_("Fact name to use for primary interface detection and hostname"), "discovery_bootif"),
+        self.set('discovery_hostname', N_("List of facts to use for the hostname (seperate by comma)"), "discovery_bootif"),
```

Comment:
@lzap Done. Test is already there ("should create discovered host without prefix").